packages/manager/modules/sms/src/sms/options/recredit/translations/Messages_fr_FR.json

```diff
@@ -2,8 +2,9 @@
   "sms_section_back_link": "Revenir aux options",
   "sms_options_recredit_title": "Recharge automatique",
   "sms_options_recredit_information_title": "Informations",
-  "sms_options_recredit_information_text": "Vous avez la possibilité de recharger automatiquement votre compte SMS lorsque votre solde franchit un certain seuil.<br/>Cette recharge, qui s'effectue automatiquement, sera prélevée sur votre compte client OVH.",
-  "sms_options_recredit_information_note": "Veuillez noter que la recharge ne sera possible que si la balance de votre compte client OVH est positive ou nulle. Il vous est possible de créditer votre compte client dans votre Espace Client Télécom section Facturation.",
+  "sms_options_recredit_information_text": "Vous avez la possibilité de recharger automatiquement votre compte SMS lorsque votre solde franchit un certain seuil.",
+  "sms_options_recredit_information_note": "L’option de recharge automatique ne peut être activée qu'à partir de 2 mois d'ancienneté de votre service. Un moyen de paiement de type Prélèvement SEPA doit également être présent et valide sur votre compte OVHcloud.",
+  "sms_options_recredit_information_note2": "Cette recharge, qui s’effectue automatiquement, sera prélevée sur votre compte client OVHcloud.",
   "sms_options_recredit_manage_title": "Gérer les options",
   "sms_options_recredit_manage_threshold_label": "Seuil minimum :",
   "sms_options_recredit_manage_threshold_text": "{{ creditThresholdForAutomaticRecredit | number:2 }} crédits",
```

Comment:
We can't use the angular filter (number:2) directly into template ?

Reply:
This part hasn't been touched (not the scope of this Pull Request).

AngularJS filters are used across multiple translation files according to following command output:
```sh
$ git grep '|' -- "**/Messages_fr_FR.json" | cut -d ':' -f 1 | uniq
packages/manager/apps/telecom/src/app/common/translations/Messages_fr_FR.json
packages/manager/modules/sms/src/sms/options/recredit/translations/Messages_fr_FR.json
packages/manager/modules/sms/src/sms/receivers/translations/Messages_fr_FR.json
packages/manager/modules/sms/src/sms/sms/hlr/translations/Messages_fr_FR.json
packages/manager/modules/sms/src/sms/sms/incoming/translations/Messages_fr_FR.json
packages/manager/modules/sms/src/sms/sms/outgoing/translations/Messages_fr_FR.json
packages/manager/modules/sms/src/sms/sms/pending/translations/Messages_fr_FR.json
```

This is something that can be addressed via its own Pull Request I think.

Please let me know if you plan to address it.
Thanks!